www/js/services.js

```diff
@@ -102,14 +100,14 @@ return {
     getResultsByFirstLetters : function(query, field, sql) {
         var query = query || '';
         var field = field || 'gurmukhi_search';
-        var sql = sql || "SELECT * FROM scriptures WHERE "+field+" LIKE '"+query+"%' LIMIT 10";
+        var sql = sql || "SELECT * FROM scriptures WHERE "+field+" LIKE '"+query+"%' LIMIT 50";
         var params = '?filter='+field + ' like '+query+'%&limit=10';
```

Comment:
Perhaps extract out into a `var LIMIT = '50'` so we can just change in one place next time!